### DISCUSSION TITLE
Ink/Stitch importer for Blender

### DISCUSSION THREAD

**Discussion by javl** [inkstitch/inkstitch#3349] [General]
After working with Ink/Stitch for a bit I was looking for a better way to visualize the result, so I made a (free) Blender extension that lets you import files generated with Ink/Stitch (technically also files created in other embroidery software, but I use it with Ink/Stitch and it uses pyembroidery in the background).

You can find the extension [over here](https://extensions.blender.org/add-ons/embroidery-importer/) on the Blender website, or you can install it straight from Blender's `preferences>extensions` menu. The GitHub repository can be found [here](https://github.com/javl/blender-embroidery).

So no issue, but a thank you for creating Ink/Stitch, and I though your users might find the addon useful.

![image](https://github.com/user-attachments/assets/f85a2df5-c9c6-48e0-b4a2-0473f63612fb)

**Comment by kaalleen** [comment]
I love Blender. This is a very interesting approach. Thank you for sharing!!

**Comment by lexelby** [comment]
WOW!! I really wish we could do something like this in Ink/Stitch itself for realistic rendering, but I don't know anything about 3d graphics.  I didn't even think of using Blender.

I wonder if we can have Ink/Stitch render the realistic preview using Blender automatically if the user has it installed?

**Comment by lexelby** [comment]
There's bpy, a python module version of blender! But it's a couple hundred megabytes so maybe that won't work ;)
